src/mongodb.appup.src

```diff
@@ -1,184 +1,95 @@
 %% -*- mode: erlang -*-
-{"3.0.17",
- [{"3.0.16", [
-    {load_module, mongoc, brutal_purge, soft_purge, []},
-    {load_module, mc_worker_api, brutal_purge, soft_purge, []},
-    {load_module, mongo_api, brutal_purge, soft_purge, []}
-  ]},
-  {"3.0.15", [
-    {load_module, mongoc, brutal_purge, soft_purge, []},
-    {load_module, mc_worker_api, brutal_purge, soft_purge, []},
-    {load_module, mongo_api, brutal_purge, soft_purge, []},
-    {load_module, mc_topology_logics, brutal_purge, soft_purge, []}
-  ]},
-  {"3.0.14", [
-    {load_module, mongoc, brutal_purge, soft_purge, []},
-    {load_module, mc_worker_api, brutal_purge, soft_purge, []},
-    {load_module, mongo_api, brutal_purge, soft_purge, []},
-    {load_module, mc_topology_logics, brutal_purge, soft_purge, []}
-  ]},
-  {"3.0.13", [
-    {load_module, mongoc, brutal_purge, soft_purge, []},
-    {load_module, mc_worker_api, brutal_purge, soft_purge, []},
-    {load_module, mongo_api, brutal_purge, soft_purge, []},
-    {load_module, mc_auth_logic, brutal_purge, soft_purge, []},
-    {load_module, mc_worker, brutal_purge, soft_purge, []},
-    {load_module, mc_topology_logics, brutal_purge, soft_purge, []},
-    {load_module, mc_util, brutal_purge, soft_purge, []}
-  ]},
-  {"3.0.12", [
-    {load_module, mongoc, brutal_purge, soft_purge, []},
-    {load_module, mc_worker_api, brutal_purge, soft_purge, []},
-    {load_module, mongo_api, brutal_purge, soft_purge, []},
-    {load_module, mc_auth_logic, brutal_purge, soft_purge, []},
-    {load_module, mc_worker, brutal_purge, soft_purge, []},
-    {load_module, mc_util, brutal_purge, soft_purge, []},
-    {load_module, mc_topology_logics, brutal_purge, soft_purge, []},
-    {load_module, mc_topology, brutal_purge, soft_purge, []}
-  ]},
-  {"3.0.11", [
+{"3.0.18", [
+   {<<"3.0.[0-7]">>, [
+    {restart_application, mongodb}
```

Comment:
Just curiosity: those versions are too old, that's why we simply restart the whole app?

Reply:
I guess so, this was the instruction for "3.0.[0-7]" before my changes so I just assumed it was the right thing to do.